src/OpenApi/sample/Shared/SharedTypes.DeeplyNestedType.cs

```diff
@@ -0,0 +1,181 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+public class DeeplyNestedLevel1
```

Comment:
Since we require all types to be public for discovery, we moved the shared types that are used by tests and the sample app to the sample app since the tests reference the sample app.

Previously, we used compile includes with internal types which doesn't work in this new model.